src/sonic-yang-mgmt/tests/libyang-python-tests/sample-yang-models/test-vlan.yang

```diff
@@ -125,9 +132,14 @@ module test-vlan {
 				leaf port {
 					/* key elements are mandatory by default */
 					mandatory true;
-					type leafref {
-						path /port:port/port:PORT/port:PORT_LIST/port:port_name;
-					}
+                                        type union {
```

Comment:
>                                         [](http://example.com/codeflow?start=0&length=40)

Could you fix the indentation of the whole file? We normally use 4 spaces instead of one tab.